### PR TITLE
fix : weekly/daily email updates fix #1224

### PR DESCRIPTION
**Pull request by BharathSubu**
### Describe what this PR does
- Fixes the weekly email update's inability to set time [Issue 1224](https://github.com/dailydotdev/daily/issues/1224)

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [x] MobileL (420px)
- [x] Tablet (656px)
- [x] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by rebelchris — APPROVED** [review]
Thanks for this fix! 
Looks good :) 

@BharathSubu Just noticed one small linting issue, if you could resolve that we can merge it later.

@sshanzel for your approval

File: packages/webapp/pages/account/notifications.tsx [hunk 1]
```diff
@@ -274,7 +274,7 @@ const AccountNotificationsPage = (): ReactElement => {
       });
     }
 
-    subscribePersonalizedDigest({ type, hour: preferredHour });
+    subscribePersonalizedDigest({ type, hour: preferredHour , sendType : personalizedDigestType as SendType  });
```
Comment (sshanzel) [inline packages/webapp/pages/account/notifications.tsx:277]: To fix the lint issue.

```suggestion
    subscribePersonalizedDigest({
      type,
      hour: preferredHour,
      sendType: personalizedDigestType as SendType,
    });
```

**Review by sshanzel — APPROVED** [review]
Looks great to me! Thank you so much for the contribution 🥳 🚀

**Comment by BharathSubu** [comment]
hey @sshanzel Should i update branch with 
update with merge commit / update with rebase

image

**Review by omBratteng — APPROVED** [review]
Looks good. Approved preview deployment. Going to test it out first.

File: packages/webapp/pages/account/notifications.tsx (hunk 1 above)
Comment (omBratteng) [reply packages/webapp/pages/account/notifications.tsx]: Linter wants it all on one line

File: packages/webapp/pages/account/notifications.tsx [hunk 2]
```diff
@@ -274,7 +274,11 @@ const AccountNotificationsPage = (): ReactElement => {
       });
     }
 
-    subscribePersonalizedDigest({ type, hour: preferredHour });
+    subscribePersonalizedDigest({
+      type,
+      hour: preferredHour,
+      sendType: personalizedDigestType as SendType,
+    });
```
Comment (omBratteng) [inline packages/webapp/pages/account/notifications.tsx:281]:
```suggestion
    subscribePersonalizedDigest({type, hour: preferredHour, sendType: personalizedDigestType as SendType});
```
Should make linter happy

Comment (sshanzel) [reply packages/webapp/pages/account/notifications.tsx:281]: Getting this unfortunately 😢 

Screenshot 2024-06-28 at 6 00 11 PM

**Comment by sshanzel** [comment]
> hey @sshanzel Should i update branch with
update with merge commit / update with rebase

Updated it now for you 🥳 
Once everything have turned green, we will merge this PR ✅

File: packages/webapp/pages/account/notifications.tsx (hunk 2 above)
Comment (omBratteng) [reply packages/webapp/pages/account/notifications.tsx:281]: Ah, maybe Vercel was going off a old commit when it said it failed because of linting.

**Comment by omBratteng** [comment]
I tested the change in preview, and it works as intended. And I see that @sshanzel fixed the test 😄

**Comment by sshanzel** [comment]
@omBratteng thank you for testing on the preview deployment. Thought you were bugged by the failing test (which was not giving any clue where it fails) 😂

**Comment by omBratteng** [comment]
> @omBratteng thank you for verifying on the preview deployment. Thought you were bugged by the failing test (which was not giving any clue where it fails) 😂

Both 😬

**Pull request merged**